contracts/Safe.sol

```diff
@@ -239,9 +239,10 @@ contract Safe is
         // solhint-disable-next-line avoid-tx-origin
         address payable receiver = refundReceiver == address(0) ? payable(tx.origin) : refundReceiver;
         if (gasToken == address(0)) {
-            // For ETH we will only adjust the gas price to not be higher than the actual used gas price
+            // For native tokens, we will only adjust the gas price to not be higher than the actually used gas price
             payment = gasUsed.add(baseGas).mul(gasPrice < tx.gasprice ? gasPrice : tx.gasprice);
-            require(receiver.send(payment), "GS011");
+            (bool refundSuccess, ) = receiver.call{value: payment}("");
```

Comment:
Any test that we could do for this? (i.e. checking that all the gas is forwarded) 

I don't think the mocking library supports this, maybe a worthy feature (gas tracking)

Reply:
I think it's possible to create a receiver contract that emits an event with `gasleft()`, and then we can check if that value is in our expected ballpark.